src/mbgl/style/style_impl.cpp

```diff
@@ -56,7 +56,7 @@ void Style::Impl::loadURL(const std::string& url_) {
 
     styleRequest = fileSource.request(Resource::style(url), [this](Response res) {
         // Once we get a fresh style, or the style is mutated, stop revalidating.
-        if (res.isFresh() || mutated) {
+        if (res.isFresh()) {
```

Comment:
If the intent here is to ensure that the cache will contain the most up-to-date style, we should probably remove this block entirely, so that sequence like the following work:

1. Map view displayed.
2. Current style is fetched. It's fresh.
3. Style is updated.

Reply:
Removed this entire code block. We'll continue to ignore updates styles once a loaded style has been mutated, but we keep up the request so that the information in the cache continues to be fresh.